assets/settings.ramsalt.local.php

```diff
@@ -29,19 +18,7 @@
 $settings['container_yamls'][] = 'sites/default/monolog.dblog.services.yml';
 
 // A default value for non-docker users' private files path.
```

Comment:
@jonsimonsen nitpick: The comment is not valid after the change anymore.

Reply:
I assume my latest commits fixes this. Feel free to merge it in if it's fine now